release notes/v0.32.0.md

```diff
@@ -0,0 +1,89 @@
+
+## Other enhancements and UX improvements
+
+- JS: `ArrayBuffer` is now supported for binary data throughout the codebase, including in WebSocket messages. [#1841](https://github.com/k6io/k6/pull/1841)
+
+  It's now possible to send binary WS messages with the `socket.sendBinary()` function and to receive binary messages with the `binaryMessage` event handler:
+
+  ```javascript
+  const binFile = open('./file.pdf', 'b');
+
+  export default function () {
+    ws.connect('http://wshost/', function(socket) {
+      socket.on('open', function() {
+        socket.sendBinary(binFile);
+      });
+
+      socket.on('binaryMessage', function(msg) {
+        // msg is an ArrayBuffer, so we can wrap it in a typed array directly.
+        new Uint8Array(msg);
+      });
+    });
+  }
+  ```
+
+  For the other backwards incompatible changes see below.
+
+- Options: K6 should now warn you on unrecognised configuration js options in most cases instead of just silently ignoring them. [#1919](https://github.com/k6io/k6/pull/1919)
+- error_code: the tag `error_code` should now be set more accurately in some cases. [#1952](https://github.com/k6io/k6/pull/1951)
+
+## Bugs fixed!
+
+- Arrival rate executors could in some cases report twice as many used VUs then what was true. [#1954](https://github.com/k6io/k6/issues/1954) fixed by [#1955](https://github.com/k6io/k6/pull/1955)
+- The newly added `responseCallback` in v0.31.0, would in cases of a response erroring mid reading the body be evaluated with the returned status code, while the reported one will be `0` as the response errored out and k6 does not return half response. Now responseCallback will receive status `0`. [#1962](https://github.com/k6io/k6/pull/1962)
+- Fix kafka output not being usable with influxdb format after v0.31.0 changes. [#1914](https://github.com/k6io/k6/pull/1914)
+- Error out with a nice message if `ramping-vus` executor would've not run a single iteration instead of just doing nothing. [#1942](https://github.com/k6io/k6/pull/1942)
+
+## Internals
+
+- JS: Way for js modules to have per VU initialization and object. This can also be used by xk6 modules by implementing `github.com/loadimpact/k6/js/modules#HasModuleInstancePerVU`. Part of [#1911](https://github.com/k6io/k6/pull/1911)
+
+
+## Breaking changes
+
+- JS: The expanded support for `ArrayBuffer`, continued from [v0.31.0](https://github.com/k6io/k6/releases/tag/v0.31.0), introduces some changes that might break current scripts that relied on the previous array of integers values returned by some of our internal APIs. Specifically these cases now return `ArrayBuffer` instead: `open(..., 'b')`, binary response bodies for requests that specified `responseType: 'binary'` (including when `http.batch()` is used), `crypto.randomBytes()`, `hasher.digest('binary')` and `encoding.b64decode()`.
```

Comment:
Yes, I know I added this :smile:, but might be good to mention since `b64decode()` previously returned a string.

```suggestion
- JS: The expanded support for `ArrayBuffer`, continued from [v0.31.0](https://github.com/k6io/k6/releases/tag/v0.31.0), introduces some changes that might break current scripts that relied on the previous array of integers or string values returned by some of our internal APIs. Specifically these cases now return `ArrayBuffer` instead: `open(..., 'b')`, binary response bodies for requests that specified `responseType: 'binary'` (including when `http.batch()` is used), `crypto.randomBytes()`, `hasher.digest('binary')` and `encoding.b64decode()`.
```